README.md

```diff
@@ -38,16 +38,16 @@ The documentation of API Platform's Client Generator can be browsed [on the offi
 
 ### Hydra
 
-    npx @api-platform/client-generator https://demo.api-platform.com/ output/ --resource Book
+    pnpm create @api-platform/client https://demo.api-platform.com/ output/ --resource Book
```

Comment:
I think more people know `npm`, we should keep it.

Reply:
It's because I switched our default package manager to `pnpm` in the distribution. I documented both PM in the docs.